src/parsers/rdp.rs

```diff
@@ -237,13 +237,21 @@ macro_rules! impl_rdp {
                 self.fail_pos = 0;
             }
 
+            fn queue_debug(&self) -> Vec<(Token<Rule>, String)> {
+                self.queue
+                    .clone()
+                    .into_iter()
+                    .map(|t| (t, self.input().slice(t.start, t.end).to_string()))
```

Comment:
It is explicit, but someone new to Rust might find it odd to see `"I am already a string".to_string()`. I still kind of do. And also the fact that most of the code already uses the `to_onwed` method makes me lean towards this one.